tasks/fix-cat2.yml

```diff
@@ -811,48 +811,55 @@
       - home
 
 - name: "MEDIUM | RHEL-07-020620 | PATCH | All local interactive user home directories defined in the /etc/passwd file must exist."
```

Comment:
Change PATCH to AUDIT

Reply:
But this one's the actual fix, not the audit, right?

Reply:
@shepdelacreme , do we want to label tasks as "PATCH" if all the task does is warn the user?

Reply:
As written, these lines for 020620 through 020650 are file tasks, not just debug warnings. Though if warnings should be flagged as AUDIT or something else, I'll make sure to add that to the sudoers PR.

Reply:
This is a good question. When we deprecated the separate AUDIT tasks we sort of did away with that nomenclature. However I don’t think we ever set a standard. For tasks that don’t make any changes I.e. debug warnings or tasks that perform checks prior to applying fix, it may be preferable to use AUDIT...but I’m open to other preferences.